### DISCUSSION TITLE
Proxy rotation and multi-proxy argument

### DISCUSSION THREAD

**Discussion by ApoNikol** [seleniumbase/SeleniumBase#3050] [Q&A] [answered]
Hi, i am currently using the sb format (with SB(uc=True, user_data_dir=dir, uc_cdp_events=True, maximize=True, do_not_track=True, proxy=proxy) as sb:),
I have a list of proxies on multithreading mode and i read the documents of https://github.com/seleniumbase/SeleniumBase for --multi-proxy  # (Allow multiple authenticated proxies when multi-threaded.)
If i pass this argument as True, can i rotate between proxies in each chrome instance?
My goal is to lower the continuing requests of one proxy by changing to another during the execution of a script.
Thank you very much!

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2860#issuecomment-2176340503

Note that once you set a proxy for a browser, you have to keep that proxy for that browser.
(Open a new browser if you want different proxy settings.)

**Comment by OpsecGuy** [comment]
A browser extension can help "dynamically" change proxy settings.

**Reply by ApoNikol** [reply]
Do you suggest any that worked for you? Thanks!

**Reply by WillCaton2350** [reply]
Something like FoxyProxy.
